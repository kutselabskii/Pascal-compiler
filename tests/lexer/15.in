123.12e-2