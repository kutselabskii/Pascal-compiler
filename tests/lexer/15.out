Line Col  Type           Value                         Original                      
1    10   Float          1.2312                        1.231200                      
1    10   End of file    EOF                                                         
